src/zjs_ocf_client.c

```diff
@@ -1093,4 +1093,23 @@ jerry_value_t zjs_ocf_client_init()
     return ocf_client;
 }
 
+void zjs_ocf_client_cleanup()
+{
+    if (resources) {
```

Comment:
Hmm, using such a generic variable name isn't a great idea, for a global. But especially when it's not defined as static - can it be?